responsibleai_text/responsibleai_text/rai_text_insights/rai_text_insights.py

```diff
@@ -637,13 +637,17 @@ def _save_predictions(self, path):
             is_classification_task)
         predict_output = self._wrapped_model.predict(
             test_without_target_column)
+        if isinstance(predict_output, pd.DataFrame):
```

Comment:
I think this is more a detail that should be handled by ml-wrappers?  The predict function should and currently is always giving numpy arrays for all of our wrapped models.

Reply:
Removed!